src/TTextEdit.h

```diff
@@ -85,11 +85,11 @@ class TTextEdit : public QWidget
     int bufferScrollUp(int lines);
     int bufferScrollDown(int lines);
     bool isTailMode();
-    void copySelectionToClipboard();
     void setConsoleFgColor(int r, int g, int b) { mFgColor = QColor(r, g, b); }
     void setConsoleBgColor(int r, int g, int b) { mBgColor = QColor(r, g, b); }
     void setIsMiniConsole() { mIsMiniConsole = true; }
-    void copySelectionToClipboardHTML();
+    // Returns whether the timestamps are showing AS A RESULT OF the call:
```

Comment:
What do you mean by 'as a result of the call'?

Reply:
This class needs to know what the time-stamp visibility setting is and it is private IIRC so passing the value back says having to change the visibility _just_ for this.

Reply:
`AS A RESULT OF the call` sounds like the call will have side-effects is what the comment sounds like to me. I think just `// Returns whether the timestamps are showing` will work, or if the function toggles _and_ returns the value, `// Toggles timestamps on/off and returns the current state``

Reply:
`// Toggles timestamps on/off and returns the current state` works for me. 😁

Reply:
☑️ Done